File: C programs/Multiplication_Table.c
// Program to make a multiplication table

#include <stdio.h>

int main( void ) {
    int number;

    // User enters number
    printf("Please enter an integer: ");
    scanf(" %d", &number);

    // Make multiplication table
    printf("\n-- Multiplication Table--\n");
    for ( int index = 1; index <= 10; index++ ) {
        printf("%d x %d = %d\n", number, index, ( number * index ) );
    }

    // Exit program
    return 0;
}

File: Python programs/Control Flow/multiplication_table.py
# Program to create a multiplication table to 10

# User enters number to multiply
number = int( input('Enter number to multiply: ') )

# Print multiplication table
print('Multiplication table...')
for index in range( 1, 10 + 1 ):
    sum = number * index
    print(number, 'x', index, '=', sum)
